game/scripts/npc/abilities/broodmother_spawn_spiderlings.txt

```diff
@@ -40,17 +40,17 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "buff_duration"                                   "2.0"
+        "buff_duration"                                   "5.0" //OAA
       }
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "spiderling_duration"                             "60.0"
+        "spiderling_duration"                             "80.0" //OAA
       }
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "70 140 210 280 630 980"
+        "damage"                                          "100 200 300 400 800 1200" //OAA
```

Comment:
Can you rescale this somehow? Damage at level 1 is fine: 100 or 120. But damage at level 4 should not be above 280 dmg because of the low cd. Also damage at OAA levels is too high. Even 980 damage is too much for a nuke that has less than 6 seconds (because of cd reduction talent). Imagine with octarine core, pure cancer... Abaddon's Myst Coil can be your guideline how to scale.